pages/01.Home/02.Contributing/docs.en.md

```diff
@@ -0,0 +1,321 @@
+
+If you wish to fix an issue, please check the comment thread to ensure that no one else is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it to avoid any accidental duplication of your effort.
+
+## Documentation Repository Structure
+The Mautic documentation repository consists of the following files and folders:
+
+The **`README.md`** file serves as the introduction and description of this repository. This file does not contain any product documentation.
+
+The **`SUMMARY.md`** file defines the menu of the documentation. If you add a new page to the documentation, you'll have to also add a new line there defining the title and the link to the file (formatted like the existing menu items).
+
+The folders in this repository are grouped together by topics. 
+ - The **`pages`** folder contains folders for each chapter in the Mautic user guide. Each folder contains one *.md* file that you can edit along with images used in this file. For example, within the *Campaigns* folder, you'll subfolders for subtopics such as *Managing Campaigns*, *Campaign Events*, etc. Each of these folders contain its own **`docs.en.md`** file and all images used in the *.md* files.
+  - Similarly, the **`plugins`** folder contains folders for each plugin that Mautic offers.
+   - The **`themes`** folder contains all the themes available for this repository. Each folder contains supporting files for a separate theme.
+
+## Style Guide
+
+For keeping contributions consistent, we encourage you to read our style guide which is a set of conventions that we follow while writing documentation for Mautic. You can find our style guide [here][style guide].
+
+ Mautic documentation is written with [Markdown markup][markup], a simple but flexible text formatting language. The Markdown files are appended with the *.md* extension.
+
+## How to contribute
+
+### Using Grav
+
+Previously Gitbook was used to manage the Mautic documentation. However, this became cumbersome to maintain and difficult for those who were not familiar with Git. After exploring various solutions, the Education Team decided that Grav would best meet the needs of the Community.
+1. Go to the [the login page][mautic-docs-login]. Create a Mautic Community login, or log in with your existing forum/website account.
+1. Send a message on the [Documentation slack channel][mautic-docs-slack] to have your account raised to 'editor' - get a Slack invite [here][mautic-slack-invite].
+1. When logged in, click the **Edit page in Grav** link on the relevant page and make your changes.
+
+### Using Github
+
+1. Fork this [GitHub][mautic-docs-github] repository.
+1. Select a file to edit on your fork.
+1. Make your commits.
+1. Open a pull request to `base fork: mautic/mautic-documentation` with `base: master`.
+1. Include and reference any [Issues][doc-issues] your Pull Request addresses.
```

Comment:
```suggestion
1. Include and reference any [Issues][doc-issues] your Pull Request addresses. Be sure to write 'closes #0001' if your PR will close an existing issue, so that they are automatically linked.
```